_assets/javascripts/components/location-search.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("deferred-js-ready", locationSearchInit);
+
+// if app-deferred loads first
+if (window.deferredJSReady) {
+  locationSearchInit();
+}
+
+function locationSearchInit() {
+  new CRDS.DistanceSorter();
+}
```

Comment:
I think we could probablyl pull this out and make it generic. Seems like this is going to get used on any page with self-specific JS that needs initialized after jquery and other things are loaded. Just a thought